AirCastingTests/Logic/BluetoothSession/DefaultStandaloneModeContollerTests.swift

```diff
@@ -20,19 +20,29 @@ final class DefaultStandaloneModeContollerTests: ACTestCase {
         Resolver.test.register { self.activeSessionProvider as ActiveMobileSessionProvidingService }
     }
     
-    func TOBEDONEtest_whenThereIsActiveSession_changesSessionStatusToDisconnected() throws {
+    func test_whenThereIsActiveSession_changesSessionStatusToDisconnected() throws {
+        activeSessionProvider.setActiveSession(session: Session.mobileAirBeamMock, device: .init(name: "Device", uuid: "1234"))
         XCTAssertNotNil(activeSessionProvider.activeSession)
         sut.moveActiveSessionToStandaloneMode()
         XCTAssertEqual([SessionStatus.DISCONNECTED], storage.callHistory)
     }
     
-    func TOBEDONEtest_whenThereIsActiveSessionWithLocation_stopLocationTracking() throws {
+    func test_whenThereIsActiveSessionWithLocation_stopLocationTracking() throws {
```

Comment:
This test fails, should it?
![Zrzut ekranu 2022-12-5 o 12 00 13](https://user-images.githubusercontent.com/44276070/205621235-29240e2a-400a-4f77-9142-f1ea7b22c110.png)